### DISCUSSION TITLE
Muse volume buttons not working with Home Assistant

### DISCUSSION THREAD

**Discussion by ralphxj** [sle118/squeezelite-esp32#269] [Q&A] [answered]
I'm running Squeezelite-esp on an ESP Muse luxe speaker. When connecting through bluetooth, everything works great. When connected to Home Assistant using Slimproto, the speaker connects and plays audio, however, when I click on either the volume up or down button on the speaker itself, it doesn't change the volume in Home Assistant and I get the following error in the Squeezelite-esp log (nothing in HA): connect_cli_socket:188 unable to connect to server 192.168.2.3:9090 with cli

What am I missing to get these buttons to work in HA, or is that not an option?

I'm running: fw: Muse.16.2.1306.master-v4.3, mode: Muse

**Comment by sle118** [comment]
The volume button locally affects the volume when playing bluetooth, as there is no server involved. Pressing any button under SLIMPROTO, on the other hand, sends a command to the server which is where handling is performed. So in order for volume control to work, you need to have an instance of the logitech media server running. Is this the case? if so, then the cli error may indicate that your muse speaker is unable to reach the server at the specified address

**Comment by philippe44** [comment, marked as answer]
You probably have a password on your LMS server. In that case, look at the README of our project and use the "raw" mode for buttons.
